lib/order.rb

```diff
@@ -0,0 +1,35 @@
+require_relative 'menu'
+require_relative 'text'
+
+class Order
+  attr_reader :current_order, :menu
+
+  def initialize(menu = Menu.new)
+    @current_order = {}
+    @menu = menu
+  end
+
+  def select_item(item,quantity)
+    if item_included?(item)
+      @current_order[item] = quantity
+    else
+      raise 'please select item from menu'
```

Comment:
Style/SignalException: Use `fail` instead of `raise` to signal exceptions. (https://github.com/bbatsov/ruby-style-guide#fail-method)